cli/parameter.go

```diff
@@ -127,3 +127,20 @@ func parseParameterMapFile(parameterFile string) (map[string]string, error) {
 	}
 	return parameterMap, nil
 }
+
+// buildDebugFlags contains options relating to troubleshooting build issues.
+type buildDebugFlags struct {
+	provisioner bool
+}
+
+func (bdf *buildDebugFlags) cliOptions() []serpent.Option {
```

Comment:
Could we instead hook into the `CODER_VERBOSE` setting?
I'm curious why we need a new setting.

Reply:
Enabling provisioner debug logs is only allowed if `--enable-terraform-debug-mode` is set. Hooking into `CODER_VERBOSE` would potentially make unrelated stuff fail if this was not set.

Reply:
I'm not a fan of piggy-backing on verbose either, it essentially changes the behavior of a build, not just the output of the CLI.